modules/nodes-nodes-audit-policy.adoc

```diff
@@ -1,9 +1,9 @@
 // Module included in the following assemblies:
 //
-// * nodes/nodes/nodes-nodes-audit-config.adoc
+// * security/audit-log-policy-config.adoc
 
 [id="configuring-audit-policy_{context}"]
-= Configuring the node audit log policy
+= Configuring the audit log policy
```

Comment:
@bergerhoffer I see that you didn't add the sentence below. Would you be able to fix it up while you are in the file? it doesn't sound quite right to me.
_when logging requests that come_ -> _when logging the requests that come_ or _when logging requests come_